### PR TITLE
SEO-173273 - WinUI weblink Color Palette/picker images

### PR DESCRIPTION
**Pull request by Sharly-Atieno**
Hi @MallikaRK ,
Worked on image alt text and renamed images.

Regards,
Sharly A.

### CODE REVIEW

File: winui/DropDown-Color-Palette/Getting-Started.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ SfDropDownColorPalette sfDropDownColorPalette =  new SfDropDownColorPalette();
 {% endhighlight %}
 {% endtabs %}
 
-![Dropdown Color Palette control with theme and standard color items](Getting-Started_images/ThemeColorPanel.png)
+![WinUI Dropdown Color Palette control with theme and standard color items](getting-started_images/winui-color-palette-theme-colo-panel.png)
```
Comment (MallikaRK) [inline winui/DropDown-Color-Palette/Getting-Started.md]: image name is wrong. check properly.
colo-panel

File: winui/DropDown-Color-Palette/Getting-Started.md [hunk 2]
```diff
@@ -146,7 +146,7 @@ SfDropDownColorPalette sfDropDownColorPalette =  new SfDropDownColorPalette();
 {% endhighlight %}
 {% endtabs %}
 
-![Dropdown Color Palette control with theme and standard color items](Getting-Started_images/ThemeColorPanel.png)
+![WinUI Dropdown Color Palette control with theme and standard color items](getting-started_images/winui-color-palette-theme-color-panel.png)
```
Comment (MallikaRK) [inline winui/DropDown-Color-Palette/Getting-Started.md:149]: Change image name under in Folder.
image

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: winui/DropDown-Color-Palette/Getting-Started.md (hunk 1 above)
Comment (Sharly-Atieno) [reply winui/DropDown-Color-Palette/Getting-Started.md]: Renamed

File: winui/DropDown-Color-Palette/Getting-Started.md (hunk 2 above)
Comment (Sharly-Atieno) [reply winui/DropDown-Color-Palette/Getting-Started.md:149]: Changed

**Review by MallikaRK — APPROVED** [review]
Approved

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**